src/plugins/condition/components/CurrentOutput.vue

```diff
@@ -22,7 +22,7 @@
 
 <template>
 <section id="current-output">
-    <div v-if="condition"
+    <div v-if="output"
```

Comment:
As currently implemented there would always be output as it defaults to false, right? I'm thinking this shouldn't be the case though, as mentioned in earlier comment. If we have it default to no output it makes sense to have this check, but there should be a v-else with appropriate instruction like "Please select condition output".

Reply:
The default output always exists. As such it will display whatever the output is for that default no matter what.